src/main/kotlin/fi/metatavu/lipsanen/projects/milestones/tasks/TaskEntity.kt

```diff
@@ -32,6 +33,22 @@ class TaskEntity : Metadata() {
     @Enumerated(EnumType.STRING)
     lateinit var status: TaskStatus
 
+    @OneToMany(mappedBy = "task", cascade = [CascadeType.ALL], orphanRemoval = true)
+    lateinit var assignees: List<TaskAssigneeEntity>
+
+    @Enumerated(EnumType.STRING)
+    @Column(nullable = true)
```

Comment:
```suggestion
    @Enumerated(EnumType.STRING)
```
@Column shoult not be neeeded